Spatial_searching/include/CGAL/Kd_tree.h

```diff
@@ -77,12 +104,15 @@ class Kd_tree {
   typedef EnablePointsCache Enable_points_cache;
 
 private:
+  
   SearchTraits traits_;
   Splitter split;
 
-
-  // wokaround for https://svn.boost.org/trac/boost/ticket/9332
-#if   (_MSC_VER == 1800) && (BOOST_VERSION == 105500)
+#if defined(CGAL_TBB_STRUCTURE_IN_KD_TREE)
+  tbb::concurrent_vector<Internal_node> internal_nodes;
+  tbb::concurrent_vector<Leaf_node> leaf_nodes;
+#elif (_MSC_VER == 1800) && (BOOST_VERSION == 105500)
```

Comment:
you should use the boost macro instead of `_MSC_VER`

Reply:
Not my code but I'll look into it.

Reply:
I can save you some time: CGAL-5.0 only supports Boost 1.57 and later. No need to test `BOOST_VERSION == 105500`.